notebooks/week6/02.refresh_monitor.py

```
@@ -0,0 +1,119 @@
+from pyspark.sql.functions import col
```

Ruff (0.7.0)

`pyspark.sql.functions.col` imported but unused

Remove unused import: `pyspark.sql.functions.col`

(F401)